src/components/app.jsx

```diff
@@ -1,9 +1,28 @@
 import React from "react";
-import Main from "./main";
+import {BrowserRouter, Switch, Route} from "react-router-dom";
 import PropTypes from "prop-types";
+import Main from "./main";
+import Login from "./login";
+import Favorites from "./favorites";
+import Property from "./property";
 
 const App = (props) => {
-  return <Main placesCount={props.placesCount} />;
+  return <BrowserRouter>
+    <Switch>
+      <Route exact path="/">
+        <Main placesCount={props.placesCount} />
+      </Route>
+      <Route path="/login">
```

Comment:
Use exact for path="", so we don't render `/login` when trying to access `/login/blabla` for example